frontend/src/lib/components/PropertyFilters/components/PropertyValue.tsx

```diff
@@ -58,6 +58,16 @@ function getValidationError(operator: PropertyOperator, value: any): string | nu
     return null
 }
 
+function parseDate(value: string | number | undefined): dayjs.Dayjs {
+    const date = dayjs(value)
+
+    if (!date.isValid() && value === '-365d') {
+        return dayjs().subtract(1, 'y')
+    }
```

Comment:
This looks very hardcoded somehow. Didn't we have any better conversion functions for these types of dates... or is that all in python?